jobs/user_add/templates/pre-start.sh.erb

```diff
@@ -11,7 +11,10 @@ set -ex
 
 <% p('users').each do |user_hash| %>
   <%  user = user_hash['name']
-      sudo = user_hash['sudo'].nil? ? true : !!user_hash['sudo']
+      groups = user_hash['groups'].nil? ? ['vcap'] : user_hash['groups']
+      groups |= ["bosh_sshers"]
+      groups |= ["admin", "bosh_sudoers"] if user_hash['sudo']
```

Comment:
Do you mind changing this into a multi-line conditional statement since this conditional is very important to keep and it's easy to overlook when it is trailing like this?